src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,55 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_CHAR_COUNT = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null) {
+            throw new RegistrationException("Login can't be null"
+                    + user.getLogin()
+                    + "Min char count is "
+                    + MIN_CHAR_COUNT);
+        }
+        if (user.getPassword() == null) {
+            throw new RegistrationException("Password can't be null"
+                    + user.getPassword()
+                    + "Min char count is "
+                    + MIN_CHAR_COUNT);
+        }
+        if (user.getAge() == null) {
```

Comment:
Again, you're checking if the user's age is null and then trying to use it in the exception message. This will lead to a null pointer exception. Remove the usage of the user's age in the exception message.